plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,794 @@
+          type: integer
+          required: false
+          example: 0
+        category:
+          title: category
+          description: Website classification and category related to the content and industry of the site. Over 70 categories are available including "Video Streaming", "Trackers", "Gaming", "Privacy", "Advertising", "Hacking", "Malicious", "Phishing", etc. The value will be "N/A" if unknown.
+          type: string
+          required: false
+          example: 'Search Engines'
+        domain_age:
+          title: Domain Age
+          description: A human description of when this domain was registered.
+          type: object
+          required: false
+          example: '{"human": "26 years ago","timestamp": 874296000,"iso": "1997-09-15T00:00:00-04:00"}'
+  phoneLookup:
+    title: Phone Reputation
+    description: Phone Reputation
+    input:
+      phone:
+        title: Phone
+        description: Phone for which information is desired
+        type: string
+        required: true
+        example: '+918765433210'
+      strictness:
+        title: Strictness
+        description: How in depth (strict) do you want this query to be? Higher values take longer to process and may provide a higher false-positive rate. We recommend starting at "0", the lowest strictness setting, and increasing to "1" or "2" depending on your levels of fraud.
+        type: integer
+        required: true
+        default: 0
+        example: 0
+        enum:
+          - 0
+          - 1
+          - 2
+      country:
+        title: Country
+        description: You can optionally provide us with the default country or countries(comma separated) this phone number is suspected to be associated with. Our system will prefer to use a country on this list for verification or will require a country to be specified in the event the phone number is less than 10 digits.
+        type: string
+        required: false
+        example: 'US'
+    output:
+      formatted:
+        title: Formatted
+        description: The phone number formatted in the international dialing code. N/A if not formattable.
+        type: string
+        required: false
+        example: '0'
+      local_format:
+        title: Local Format
+        description: The phone number formatted in the country's local routing rules with area code. N/A if not formattable.
+        type: string
+        required: false
+        example: '0'
+      valid:
+        title: Valid
+        description: Is the phone number properly formatted and considered valid based on assigned phone numbers available to carriers in that country?.
+        type: boolean
+        required: false
+        example: false
+      fraud_score:
+        title: Fraud Score
+        description: The IPQS risk score which estimates how likely a phone number is to be fraudulent. Scores 85+ are high risk.
```

Comment:
Please remove all the dots `.` from every description. Also please include in the comment results after running `insightconnect-validators` on it.

Reply:
Hi @igorski-r7 
We are facing issue with insightconnect-validators while installing using `pip install insightconnect-integrations-validators`.
Can you please guide us on this.